core/src/com/unciv/logic/civilization/CivilizationInfo.kt

```diff
@@ -401,9 +394,11 @@ class CivilizationInfo {
         val cityStateLocation = if (cities.isEmpty()) null else getCapital().location
 
         val giftAmount = Stats(gold = 15f)
+        val faithAmount = Stats(faith = 15f)
```

Comment:
Why is this seperate from the giftAmount? They are used for the exact same thing and should also be doubled when meeting them for the first time. Adding them to the existing stats when the city state is religious should suffice

Reply:
It is separate because if i send a notification, it will print 15 gold

Reply:
If you add the faith to it, it will print '15 gold, 15 faith'. It should also only be 4 faith iirc

Reply:
This hasn't been adressed yet.